examples/simple-node/build.gradle

```diff
@@ -1,10 +1,5 @@
 apply plugin: 'com.github.node-gradle.node'
 
-node {
```

Comment:
I just removed it to avoid installing Node.js two more times during the build. But I can at least let this block with default configuration. You are right, as this is an example, we should show how to configure it.

Reply:
I am going to change that.

Reply:
download = false is a legit line